bentoml/_internal/frameworks/pytorch_lightning.py

```diff
@@ -156,72 +103,24 @@ def configure_optimizers(self):
-    """  # noqa
-    context: t.Dict[str, t.Any] = {
-        "framework_name": "torch",
-        "pip_dependencies": [
-            f"torch=={get_pkg_version('torch')}",
-            f"pytorch_lightning=={get_pkg_version('pytorch_lightning')}",
-        ],
-    }
-
-    with bentoml.models.create(
-        name,
-        module=MODULE_NAME,
-        options=None,
-        context=context,
-        labels=labels,
-        custom_objects=custom_objects,
-        metadata=metadata,
-    ) as _model:
-
-        weight_file = _model.path_of(f"{SAVE_NAMESPACE}{PT_EXT}")
-        _model.info.context["model_format"] = "pytorch_lightning:v1"
-        torch.jit.save(model.to_torchscript(), weight_file)  # type: ignore[reportUnknownMemberType]
-
-        return _model.tag
-
-
-class _PyTorchLightningRunner(BasePyTorchRunner):
-    def _load_model(self):
-        return load(self._tag, device_id=self._device_id, model_store=self.model_store)
-
-
-@inject
-def load_runner(
-    tag: t.Union[str, Tag],
-    *,
-    predict_fn_name: str = "__call__",
-    partial_kwargs: t.Optional[t.Dict[str, t.Any]] = None,
-    name: t.Optional[str] = None,
-) -> "_PyTorchLightningRunner":
     """
-    Runner represents a unit of serving logic that can be scaled horizontally to
-    maximize throughput. :func:`bentoml.pytorch_lightning.load_runner` implements a Runner class that
-    wrap around a :obj:`pl.LightningModule` instance, which optimize it for the BentoML runtime.
-
-    Args:
-        tag (:code:`Union[str, Tag]`):
-            Tag of a saved model in BentoML local modelstore.
-        predict_fn_name (:code:`str`, default to :code:`__call__`):
-            inference function to be used.
-        partial_kwargs (:code:`Dict[str, Any]`, `optional`,  default to :code:`None`):
-            Common kwargs passed to model for this runner
-
-    Returns:
-        :obj:`~bentoml._internal.runner.Runner`: Runner instances for :mod:`bentoml.pytorch_lightning` model
+    if not isinstance(model, pl.LightningModule):
+        raise TypeError(
+            f"`model` must be an instance of `pl.LightningModule`, got {type(model)}"
+        )
 
-    Examples:
+    script_module = model.to_torchscript()
 
-    .. code-block:: python
+    assert not isinstance(
+        script_module, dict
+    ), "Saving a dict of pytorch_lightning Module into one BentoModel is not supported"
 
-        import bentoml.pytorch_lightning
-        runner = bentoml.pytorch_lightning.load_runner("lit_classifier:20201012_DE43A2")
-        runner.run(pd.DataFrame("/path/to/csv"))
-    """
-    return _PyTorchLightningRunner(
-        tag=tag,
-        predict_fn_name=predict_fn_name,
-        name=name,
-        partial_kwargs=partial_kwargs,
+    return script_save_model(
+        name,
+        script_module,
+        signatures=signatures,
+        labels=labels,
+        custom_objects=custom_objects,
+        metadata=metadata,
+        _include_pytorch_lightning_version=True,
```

Comment:
hmm this seems to be like an internal arg from torchscript. 🤔

Reply:
We don't want to expose this arg to users, I think.

Reply:
agree.